src/fdk_rdf_parser/fdk_rdf_parser.py

```diff
@@ -11,9 +11,9 @@
 from .reference_data import extendDatasetWithReferenceData, getAllReferenceData
 
 
-def isTypeDataset(graph: Graph, topic: URIRef) -> bool:
+def isType(name: str, graph: Graph, topic: URIRef) -> bool:
```

Comment:
Endre navn til isDcatType i så fall

eventuelt ta inn en 'type: URIRef' i stedet for 'name: str' og send inn dcatURI("DataService") dcatURI("Dataset")

Reply:
God ide!